specs/_build.ham

```diff
@@ -0,0 +1,6 @@
+if [ SubDirOnce TOP specs  ] = 1 { return ; }
+
+# Recursively get all .sh files
+local SH_FILES = [ FGristFiles [ ListFilesR [ FDirName $(SUBDIR) toolsets ] : setup-toolset.sh ] ] ;
```

Comment:
Ideally we'd add a maxDepth parameter to ListFilesR, cause we really dont want it to go in the sub-sub dir. Not an issue here because the specs folder is shallow.